hack/tidy-check.sh

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 Antrea Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set +e
+
+THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null 2>&1 && pwd)"
+
+pushd "$(dirname "$THIS_DIR")" >/dev/null || exit
+
+
+PROJECT_RELATIVE_DIR=${1:-.}
+PROJECT_DIR=$(dirname "$THIS_DIR")/$PROJECT_RELATIVE_DIR
+
+MOD_FILE="$PROJECT_DIR/go.mod"
+SUM_FILE="$PROJECT_DIR/go.sum"
+TMP_DIR="$THIS_DIR/.tmp.tidy-check"
+TMP_MOD_FILE="$TMP_DIR/go.mod"
+TMP_SUM_FILE="$TMP_DIR/go.sum"
+
+# if Go environment variable is set, use it as it is, otherwise default to "go"
+: "${GO:=go}"
+
+TIDY_COMMAND="cd $PROJECT_RELATIVE_DIR && $GO mod tidy >> /dev/null 2>&1"
+
+function echoerr {
+  >&2 echo "$@"
+}
+
+function general_help {
+  echoerr "Please run the following command to generate a new go.mod & go.sum:"
+  if [ -n "$GO" ]; then
+    echoerr "  \$ make tidy"
+  else
+    echoerr "  \$ make docker-tidy"
```

Comment:
Seems that there is no target `docker-tidy` in Makefile?

Reply:
Removed, thanks.